public/content/lessons/2017/chain-rule-and-product-rule/index.mdx

```diff
@@ -7,4 +7,355 @@ video: YG15m2VwSjA
+
+What I can offer, and what I hope I have offered, is to show you where these rules come from, to show that they’re not just something to be memorized and hammered away; but instead are natural patterns that you too could have discovered by just patiently thinking through what a derivative means.
+
+In the next lesson, I’ll be talking about related rates.
+
+## Exercises
+
+### $\sin(x)^2$
+
+As a fun exercise, think about the derivative of $\sin(x)^2$. First, use the chain rule, thinking of this as shoving the function $\sin(x)$ into the function $x^2$, then taking the derivative of the outside multiplied by the derivative of the inside.
+
+Then, think of it using the product rule, interpreting it as $\sin(x) \cdot \sin(x)$, and think about how this relates to the visual for the derivative of $x^2$ shown in the last video. That should give a deeper feel for the chain rule.
+
+<Accordion title="Chain Rule Explanation">
+
+Setup three numberlines to represent the function composition and nudge the input $x$ by a small amount $dx$. Remember we are interested in finding $df$, the change to the function represented on the bottom number line.
+
+<Figure
+  image="figures/exercise-1-chain-rule-setup.svg"
+/>
+
+Substitute $h=\sin(x)$ into the function composition. This simplifies $(\sin (x))^{2} = (h)^{2}$ and we know the change of the function $h^2$ given some small nudge $dh$ is $2h \cdot dh$.
+
+<Figure
+  image="figures/exercise-1-chain-rule-substitution.svg"
+/>
+
+Unfold the composition and observe that $dh = \cos(x) \cdot dx$, because we know that the change in sine is $d(\sin(x)) = \cos(x) \cdot dx$.
+
+<Figure
+  image="figures/exercise-1-chain-rule-unfolding.svg"
+/>
+
+This gives us that the change to the function is $df = 2 \sin(x) \cdot \cos(x) dx$ which we can rearrange for the derivative of the function.
+
+$$
+\frac{df}{dx} = 2 \sin(x) \cos(x)
+$$
+ 
+</Accordion>
+
+<Accordion title="Product Rule Explanation">
+
+To model the derivative of this function start by visualizing the function as the area of the square with side lengths $\sin(x)$.
+
+<Figure
+  image="figures/exercise-1-product-rule-area.svg"
+/>
+
+Then, imagine nudging the input of the function $x$ by a small amount $dx$. This introduces three new rectangles of area.
+
+<Figure
+  image="figures/exercise-1-product-rule-area-df.svg"
+/>
+
+From past chapters we know that the expression $d(\sin(x))$ is $\cos(x) \cdot dx$. Using this information we can express the chang ein area of the function as shown below. As before, the small rectangle's area in the bottom right corner is negligible.
+
+<Figure
+  image="figures/exercise-1-product-rule-derivative.svg"
+/>
+
```

Comment:
This seems like a nice place to mention explicitly that both methods give the same result. Seeing the same result is the point of doing the same problem using two different methods, so it seems worth mentioning.